plugins/processors/template/template.go

```diff
@@ -0,0 +1,38 @@
+package template
+
+import (
+	"github.com/influxdata/telegraf"
+	"strings"
+	"text/template"
+)
+
+type TemplateProcessor struct {
+	Tag      string `toml:"tag"`
+	Template string `toml:"template"`
+}
+
+func (r *TemplateProcessor) Apply(in ...telegraf.Metric) []telegraf.Metric {
+	// create template
+	tmpl, err := template.New("test").Parse(r.Template)
```

Comment:
Since we only need to do this once, let's move it into a Init() function.  This gets called by Telegraf on startup, in your tests you will need to do it manually:
```
func (r *TemplateProcessor) Init() error {
}
```